src/components/nano/OnChainBlueprintsTable.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import SortableTable from '../SortableTable';
+import EllipsiCell from '../EllipsiCell';
+import dateFormatter from '../../utils/date';
+
+// XXX We should use function component with SortableTable as a component
+// but renderTableHead and renderTableBody are implemented and not
+// expected as a props, so it demands a bigger refactor
+class OnChainBlueprintsTable extends SortableTable {
```

Comment:
suggestion(non-blocking): Maybe it shouldn't extend SortableTable but use it

Reply:
We need to extend it because we need to override the `renderTableBody` and `renderTableHead` methods. They should have been designed as props, so we could just use the component with the props defined but it wasn't.

That's the comment in the file, we should refactor it to receive these methods as props and change all the inheritance to use it.